app/build.gradle

```diff
@@ -121,6 +124,9 @@ dependencies {
     implementation 'com.davemorrissey.labs:subsampling-scale-image-view-androidx:3.10.0'
     implementation 'com.github.MikeOrtiz:TouchImageView:3.1.1'
 
+    //DO NOT REMOVE THIS UNTIL filepicker is updated to latest version in arklib 0.3.2
+    implementation 'dev.arkbuilders:arkfilepicker:0.1.2-snapshot-04'
```

Comment:
Should remove after upgrading `filepicker` to latest version in arklib 0.3.2.

Reply:
But the `ark-filepicker` doesn't import `arklib-android`, it is the opposite way around.

Reply:
v0.3.3 https://github.com/ARK-Builders/arklib-android/actions/runs/6518877888